### ISSUE TITLE
I can't create t-address

### ISSUE THREAD

**Issue by Mandoresher** [ZcashFoundation/zecwallet#327]
Can't create t-address. Button doesn't work. I push it but nothing happens

**Comment by Mandoresher** [comment]
Zecwallet FullNode desktop application

**Comment by POLANOS** [comment]
Same problem... Zecwallet Fullnode v1.7.5
Windows 10 Pro x64 21H1 - 19043.1266
debug.log: ERROR rustzcash::address_ffi: Zcash Sapling addresses are not supported
---
Can't create t-address. Button doesn't work. I push it but nothing happens

**Comment by adityapk00** [comment]
This seems like an issue with the underlying zcashd, please track progress here: https://github.com/zcash/zcash/issues/5338

**Comment by eloo0073** [comment]
Same issue... Fullnode v1.7.5...  Can not create transparent addresses.  Do we really need them anyway ? ;-)

**Comment by krychla1** [comment]
Hi @adityapk00 
now that zcash implemented the listaddresses endpoint (4.5.1-1), when will it be ready in zecwallet?

**Comment by yuriy77k** [comment]
@adityapk00 when an issue with t-addresses will be fixed?

**Comment by wbaby** [comment]
> This seems like an issue with the underlying zcashd, please track progress here: [zcash/zcash#5338](https://github.com/zcash/zcash/issues/5338)

Hi, wish an update to create t-address

**Comment by krychla1** [comment]
This is dead, I personally switched away from zcash seeing this pace of bug fixing in their solutions.

**Comment by steven-ecc** [comment]
The fix required is in ZecWallet, not in the zcashd full node. We added the needed RPC support in 4.5.1-1.

**Comment by krychla1** [comment]
Most of regular users don't use CLI for transactions. ZecWallet is mentioned as the only UI frontend in the official webpage. 
@steven-ecc, I appreciate your work on the node itself, however if you (Zcash community) want to get wider adoption you have to make things smooth also from the UI perspective. 

This issue completely blocks users from using ZecWallet with t-addresses and even after 2 months of your fix there is nothing new here. It says a lot about this project.

**Comment by wbaby** [comment]
> Most of regular users don't use CLI for transactions. ZecWallet is mentioned as the only UI frontend in the official webpage. @steven-ecc, I appreciate your work on the node itself, however if you (Zcash community) want to get wider adoption you have to make things smooth also from the UI perspective.
> 
> This issue completely blocks usage of ZecWallet with t-addresses and even after 2 months of your fix there is nothing new here. It says a lot about this project.

Totally agree with you. 

@steven-ecc  , Users need friendly UI, CLI is too hard for most of the users.

**Comment by steven-ecc** [comment]
I agree @wbaby and @krychla1 regarding a friendly UI. on the [z.cash/wallets](https://z.cash/wallets/) page, we have a few listed that you could try out. Some are multi-coin and NightHawk is a ZEC-only wallet.

**Comment by noamchom** [comment]
This t-address issue is still present in Zec Wallet Full node v1.7.6 (on Windows 10 PRO) the "New Transparent Address" button appears to take no action.

**Comment by POLANOS** [comment]
Same problem... Zecwallet Fullnode v1.7.7
Windows 10 Pro x64 21H2 - 19044.1645
===
Can't create t-address.
Button "New Transparent Address" doesn't work.
I push it but nothing happens.
I don't see the list of created t-addresses in the application and therefore I can't use them.
===
Please help!!!
